### DISCUSSION TITLE
Forms noscript message

### DISCUSSION THREAD

**Discussion by grich89** [flyntwp/flynt#372] [Q&A] [answered]
Hello. Thanks for developing an awesome and easy to use theme. I'm wondering if you have a recommendation for a form plugin or implementation strategy with Flynt? I have tried to use several plugins - HTML Forms, Forminator, and WP Forms Lite - and all three are producing a similar noscript error message: "Please enable Javascript in your browser." I'm a little confused as Javascript is enabled in my browser. I thought the error message might be related to a conflict with jQuery, but after investigating that a bit and seeing that you've already written a workaround to remove jQuery from the head element I am thinking it's something else.

**Comment by grich89** [comment]
I just wanted to follow up again to let you know I've just tested this on a fresh installation of the Flynt theme (v. 1.4) to ensure there were no issues with any of the changes I had made to it, and I also tested the Gravity Forms and Contact Form 7 plugins. Same issue, either a "Please enable Javascript in your browser" error appears or the form fields/submit button do not appear. Any guidance is appreciated, thank you!

**Comment by grich89** [comment, marked as answer]
I just figured out a solution here. The issue seemed to be a conflict between the form shortcodes and the Wysiwyg component, so I am just going to create a custom component for only the form embed code.

**Reply by steffenbew** [reply]
Happy you found a solution! We're typically working with [Contact Form 7](https://flyntwp.com/components/form-contact-form-7/) and Gravity Forms. We also prefer building dedicated form components.
